openid-federation-1_0.xml

```diff
@@ -7746,7 +7779,7 @@ HTTP/1.1 302 Found
 
       <section anchor="ClaimsRegistry" title="JSON Web Token Claims Registration">
 	<t>
-	  This specification registers the following claim in
+	  This specification registers the following claims in
```

Comment:
You raise a really good point.  We tried to register the claims used by JWTs defined by the specification that were currently unregistered.  But arguably the most important JWT defined by the spec - the Entity Statement - doesn't have all of its claims registered.  Thanks for pointing out what should have been obvious!

A quick skim makes me believe that these are such claims:  "jwks", "authority_hints", "metadata", "metadata_policy", "constraints", "metadata_policy_crit", "trust_marks", "trust_mark_issuers", "trust_mark_owners", "source_endpoint".

Of these, clearly "jwks", "metadata", and "constraints" should have broadly-usable definitions appearing in the list at https://openid.net/specs/openid-federation-1_0.html#name-general-purpose-jwt-claims.  My instinct for the rest is to register them as used by OpenID Federation but not generically.  This would parallel the way that "sub_jwk" is registered.

Do you agree with that course of action, Brian (or at least agree with it enough to review a PR on the topic)?  Others, please weigh in too!

Reply:
OK, the [latest commit](https://github.com/openid/federation/pull/51/commits/fa8a94a23e875c31537a546e37c2deaa046f297e) added registrations for the previously unregistered claims.  I shuffled the order of some of the claim registrations so that the referenced section numbers now occur in ascending order.

I believe that means that we're ready for you to re-review, @bc-pi.  Thanks!